packages/eth2.0-params/src/presets/mainnet.ts

```diff
@@ -34,23 +34,23 @@ export const GENESIS_START_SHARD = 0;
 
 // Time parameters
 export const SECONDS_PER_SLOT = 12;
-export const MIN_ATTESTATION_INCLUSION_DELAY = 2 ** 0; // slots || 12 seconds
-export const SLOTS_PER_EPOCH = 2 ** 5; // slots || 6.4 minutes
-export const MIN_SEED_LOOKAHEAD = 2 ** 0; // epochs || 6.4 minutes
-export const MAX_SEED_LOOKAHEAD = 2 ** 2; // epochs || 25.6 minutes
-export const SLOTS_PER_ETH1_VOTING_PERIOD = 2 ** 10; // slots || ~1.7 hours
-export const ETH1_FOLLOW_DISTANCE = 2 ** 10; // blocks || ~4 hours
-export const SLOTS_PER_HISTORICAL_ROOT = 2 ** 13; // slots || ~13 hours
-export const MIN_VALIDATOR_WITHDRAWAL_DELAY = 2 ** 8; // epochs || ~27 hours
-export const PERSISTENT_COMMITTEE_PERIOD = 2 ** 11; // epochs || 9 days
+export const MIN_ATTESTATION_INCLUSION_DELAY = 1; // slots || 12 seconds
+export const SLOTS_PER_EPOCH = 32; // slots || 6.4 minutes
+export const MIN_SEED_LOOKAHEAD = 1; // epochs || 6.4 minutes
+export const MAX_SEED_LOOKAHEAD = 4; // epochs || 25.6 minutes
+export const SLOTS_PER_ETH1_VOTING_PERIOD = 1024; // slots || ~1.7 hours
+export const ETH1_FOLLOW_DISTANCE = 1024; // blocks || ~4 hours
+export const SLOTS_PER_HISTORICAL_ROOT = 8192; // slots || ~13 hours
+export const MIN_VALIDATOR_WITHDRAWAL_DELAY = 256; // epochs || ~27 hours
+export const PERSISTENT_COMMITTEE_PERIOD = 2048; // epochs || 9 days
 
 export const MIN_EPOCHS_TO_INACTIVITY_PENALTY = 2 ** 2; // 25.6 minutes
 
 // State list lengths
 export const EPOCHS_PER_HISTORICAL_VECTOR = 2 ** 16;
 export const EPOCHS_PER_SLASHINGS_VECTOR = 2 ** 13;
 export const HISTORICAL_ROOTS_LIMIT = 2 ** 24;
-export const VALIDATOR_REGISTRY_LIMIT = 2 ** 40;
+export const VALIDATOR_REGISTRY_LIMIT = 1099511627776;
```

Comment:
Can we heep these params the way they were? I don't think this is much better (I think things like eg: `2 ** 40` are more readable).

Reply:
It's hard to compare them with spec configs, it's probably best to keep same notation as spec file config. There is another reason for that, tersec (webpack minimizer) has a bug with those notation and bigint. For example  2n**0n*BigInt(1e9) will get minifed to 1*BigInt(1e9) which will throw error in chrome. It's fixed in latest version but it's far from being included in webpack:
https://github.com/terser/terser/issues/546

Reply:
ok fair enough, maybe one day we can just ingest the upstream configuration files

Reply:
Yeah we had issue, to automatically generate params from: https://github.com/ethereum/eth2.0-specs/blob/dev/configs/mainnet.yaml but it got closed by stalebot :(